include/zephyr/bluetooth/cs.h

```diff
@@ -0,0 +1,86 @@
+/** @file
+ *  @brief Bluetooth Channel Sounding handling
+ */
+
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#ifndef ZEPHYR_INCLUDE_BLUETOOTH_CS_H_
+#define ZEPHYR_INCLUDE_BLUETOOTH_CS_H_
+
+/**
+ * @brief Channel Sounding (CS)
+ * @defgroup bt_cs Channel Sounding (CS)
+ * @ingroup bluetooth
+ * @{
+ */
+
+#include <stdint.h>
+#include <stdbool.h>
+
+#include <zephyr/bluetooth/hci_types.h>
+#include <zephyr/bluetooth/conn.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+enum bt_cs_sync_antenna_selection_opt {
+	/** Use antenna identifier 1 for CS_SYNC packets. */
+	BT_CS_ANTENNA_SELECTION_OPT_ONE = BT_HCI_OP_LE_CS_ANTENNA_SEL_ONE,
+	/** Use antenna identifier 2 for CS_SYNC packets. */
+	BT_CS_ANTENNA_SELECTION_OPT_TWO = BT_HCI_OP_LE_CS_ANTENNA_SEL_TWO,
+	/** Use antenna identifier 3 for CS_SYNC packets. */
+	BT_CS_ANTENNA_SELECTION_OPT_THREE = BT_HCI_OP_LE_CS_ANTENNA_SEL_THREE,
+	/** Use antenna identifier 4 for CS_SYNC packets. */
+	BT_CS_ANTENNA_SELECTION_OPT_FOUR = BT_HCI_OP_LE_CS_ANTENNA_SEL_FOUR,
+	/** Use antennas in repetitive order from 1 to 4 for CS_SYNC packets. */
+	BT_CS_ANTENNA_SELECTION_OPT_REPETITIVE = BT_HCI_OP_LE_CS_ANTENNA_SEL_REP,
+	/** No recommendation for local controller antenna selection. */
+	BT_CS_ANTENNA_SELECTION_OPT_NO_RECOMMENDATION = BT_HCI_OP_LE_CS_ANTENNA_SEL_NONE,
+};
+
+/** Connection subrating parameters for LE connections */
```

Comment:
noticed this now

suggest copying some text from the core spec, like "default CS
settings in the local Controller"